aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -752,39 +861,31 @@ async def create_vp(
     Return:
         VerifiablePresentation
     """
+    document_loader = profile.context.inject(DocumentLoader)
     req = await make_requirement(
         srs=pd.submission_requirements, descriptors=pd.input_descriptors
     )
-    result = await apply_requirements(req=req, credentials=credentials)
+    result = await apply_requirements(
+        req=req, credentials=credentials, profile=profile, suite=derive_suite
+    )
     applicable_creds, descriptor_maps = await merge(result)
     # convert list of verifiable credentials to list to dict
-    applicable_cred_dict = []
+    applicable_creds_list = []
     for credential in applicable_creds:
-        applicable_cred_dict.append(json.loads(credential.value))
+        applicable_creds_list.append(json.loads(credential.cred_value))
     # submission_property
     submission_property = PresentationSubmission(
         _id=str(uuid4()), definition_id=pd._id, descriptor_maps=descriptor_maps
     )
-
-    # defaultVPContext
-    default_vp_context = [
-        CREDENTIALS_CONTEXT_V1_URL,
-        PRESENTATION_SUBMISSION_JSONLD_CONTEXT,
-    ]
-    # defaultVPType
-    default_vp_type = [
-        VERIFIABLE_PRESENTATION_TYPE,
-        PRESENTATION_SUBMISSION_JSONLD_TYPE,
-    ]
-
-    vp = VerifiablePresentation(
-        _id=str(uuid4()),
-        contexts=default_vp_context,
-        types=default_vp_type,
-        credentials=applicable_cred_dict,
-        presentation_submission=submission_property,
+    vp = await create_presentation(credentials=applicable_creds_list)
+    vp["presentation_submission"] = submission_property.serialize()
+    signed_vp = await sign_presentation(
```

Comment:
Where is the `PRESENTATION_SUBMISSION_JSONLD_CONTEXT ` and `PRESENTATION_SUBMISSION_JSONLD_TYPE` set now?